packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -218,6 +359,18 @@ export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, {}> {
     }
   }
 
+  @autobind
+  private _useInput() {
+    if (this.props.createGenericItem) {
+      this.props.createGenericItem();
```

Comment:
this feels more like it should be `forceResolve` or something like that.

Reply:
yes, that sounds like a much better name, I'll change that where applicable